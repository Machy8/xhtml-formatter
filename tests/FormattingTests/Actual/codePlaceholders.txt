<style>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</style>

<unskippedElement>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</unskippedElement>

<script>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</script>

<script>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</script>

<div>
	<div>
		<formatter-off>
			div
				h1 Indented block in macdom
		</formatter-off>
	</div>
</div>


<div data-math="10 > 5"></div>
<div data-math='10 > 5'></div>
<style>h1 {color:red;}</style><style>h1 {color:blue;}</style>
<div class="<?php echo $classes ?>"></div>
Some text and <script script type="text/javascript" src=" somescript.js"></script>