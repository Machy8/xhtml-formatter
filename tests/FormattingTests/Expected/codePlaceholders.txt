<style>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</style>
<script>
		<div></div><div></div><div></div>
    	<?php echo 'HelloWorld' ?><div></div>
</script>
<div>
	<div>			
			div
				h1 Indented block in macdom
	</div>
</div>
<div data-math="10 > 5"></div>
<div data-math='10 > 5'></div>
<style>
	h1 {color:red;}
</style>
<style>
	h1 {color:blue;}
</style>
<div class="<?php echo $classes ?>"></div>
Some text and
<script script type="text/javascript" src=" somescript.js"></script>
